reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -279,6 +302,9 @@ def launch_refarch_env(region=None,
                     containerized,
                     s3_bucket_name,
                     s3_username,
+                    github_client_id,
+                    github_client_secret,
+                    str(map(lambda x: x.encode('ascii'), github_organization)).replace("'", '"').replace(' ', ''),
```

Comment:
looks like it prints the var as expected github_organization=["openshift","RHsyseng"]' will make the change